bin/ch-test

```diff
@@ -900,7 +900,10 @@ if [[ -z $CHTEST_INSTALLED ]]; then
     if [[ $ch_version != "$cf_version" ]]; then
         warning "inconsistent configure version: ${cf_version}"
     fi
-    src_version=$("${ch_base}/misc/version")
+    src_version=$("${ch_base}/misc/version" 2> /dev/null || echo '')
+    if [[ $src_version = '' ]]; then
+       src_version=$(cat "${ch_base}/VERSION")
+    fi
```

Comment:
I'd recommend checking for the availability of Git directly, if that's what you're after. As written, this uses `VERSION` no matter why `misc/version` failed.

Maybe that's what you want? If so, then `|| true` would be more idiomatic, and a commend explaining why it's what you want would be good.